lucille-plugins/lucille-pinecone/src/main/java/com/kmwllc/lucille/pinecone/indexer/PineconeIndexer.java

```diff
@@ -70,15 +101,16 @@ protected void sendToIndex(List<Document> documents) throws IndexerException {
         uploadDocuments(documents, (String) entry.getValue(), entry.getKey());
       }
     } else {
-      uploadDocuments(documents, defaultEmbeddingField, "");
+      uploadDocuments(documents, defaultEmbeddingField, "default");
```

Comment:
adding "default" as the default namespaces to allow delete by prefix to work. Index.list requires a non empty string for namespace to list vectors by prefix -- an operation required and recommended by pinecone to delete by prefix.
More info here: https://docs.pinecone.io/guides/data/manage-rag-documents#delete-all-records-for-a-parent-document